code401Challenges/src/main/java/code401Challenges/linkedlist/LinkedList.java

```diff
@@ -172,5 +172,31 @@ public static LinkedList mergeLists(LinkedList a, LinkedList b) {
         return b;
     }
 
+    // Merge two linked lists
+    // Received help with logic from Michelle Ferreirae
+    public static LinkedList mergeLinkedLists(LinkedList a, LinkedList b) {
+        if (a.head == null) {
+            return b;
+        }
```

Comment:
This is a good check (if list a is empty, then just return the other list). However, if list b is empty the algorithm will break. So you could add another check for if list b is empty, return a.